rfc/cdevents-spinnaker.md

```diff
@@ -0,0 +1,106 @@
+# Implementing CDEvents to Spinnaker using Java SDK
+
+
+| | |
+|-|-|
+| **Status**     | _**Proposed**, Accepted, Implemented, Obsolete_ |
+| **RFC #**      | https://github.com/spinnaker/governance/pull/299 |
+| **Author(s)**  | _Jalander Ramagiri (`@rjalander`)_|
+| **SIG / WG**   | CDF-CDEvents, CDF-SIGEvents |
+
+## Overview
+
+Many organisations use more than one CI/CD tools as part of their pipelines and most of the times the organisations implement their own glue code in order to make the tools work with each other.
+
+Implementing [CDEvents](https://cdevents.dev/) to Spinnaker enables Spinnaker to communicate with other CI/CD tools with in the ecosystem. [CDEvents](https://cdevents.dev/) is a common specification for Continuous Delivery events based on CloudEvents.
```

Comment:
I like the goal here! Having a standardized reactive system for CD is a great design goal!